tools/acrn-crashlog/README.rst

```diff
@@ -61,6 +61,96 @@ To install the build:
    $ cd acrn-crashlog
    $ sudo make install
 
+Enabling/Disabling
+******************
+
+To enable this tool:
+
+.. code-block:: none
+
+   $ sudo crashlogctl enable
+
+Then it will show:
+
+.. code-block:: none
+
+   ... Backup core pattern to /var/log/crashlog/default_core_pattern
+   '/usr/share/defaults/telemetrics/telemetrics.conf' ->
+   '/etc/telemetrics/telemetrics.conf'
+   ... Set server_delivery_enabled=false in /etc/telemetrics/telemetrics.conf
+   ... Set record_retention_enabled=true in /etc/telemetrics/telemetrics.conf
+   '/usr/share/acrn/crashlog/40-watchdog.conf' ->
+   '/etc/systemd/system.conf.d/40-watchdog.conf'
+   '/usr/share/acrn/crashlog/80-coredump.conf' ->
+   '/etc/sysctl.d/80-coredump.conf'
+   Created symlink /etc/systemd/system/hprobe.timer → /dev/null.
+   Created symlink /etc/systemd/system/telemd-update-trigger.service →
+   /dev/null.
+   Created symlink /etc/systemd/system/pstore-clean.service → /dev/null.
+   Created symlink /etc/systemd/system/pstore-probe.service → /dev/null.
+   Created symlink /etc/systemd/system/oops-probe.service → /dev/null.
+   Created symlink /etc/systemd/system/klogscanner.service → /dev/null.
+   Created symlink /etc/systemd/system/journal-probe.service → /dev/null.
+   Created symlink /etc/systemd/system/bert-probe.service → /dev/null.
+   Created symlink /etc/systemd/system/multi-user.target.wants/acrnprobe.service
+   → /usr/lib/systemd/system/acrnprobe.service.
+   Created symlink /etc/systemd/system/multi-user.target.wants/usercrash.service
+   → /usr/lib/systemd/system/usercrash.service.
+   *** Please reboot your system. ***
+
+Follow the hints to reboot the system:
+
+.. code-block:: none
+
+   $ sudo reboot
+
+To disable this tool:
+
+.. code-block:: none
+
+   $ sudo crashlogctl disable
+
+Then it will show:
```

Comment:
Same question about using `console` instead of `none` (@dbkinder )

Reply:
Yes here too.  Use ``none`` or ``bash`` for input screens, and ``console`` for output screens.

Reply:
Thanks a lot for your guide. Patch has been updated. Please help to review.